src/components/Fund/FundsList.js

```diff
@@ -0,0 +1,153 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { get, toString } from 'lodash';
+import {
+  SearchAndSort,
+  makeQueryFunction,
+} from '@folio/stripes/smart-components';
+
+import {
+  FUNDS_API,
+} from '../../common/const';
+import FinanceNavigation from '../../common/FinanceNavigation';
+
+import transitionToParams from '../../Utils/transitionToParams';
+import removeQueryParam from '../../Utils/removeQueryParam';
+import packageInfo from '../../../package';
+import { Filters, SearchableIndexes } from './fundFilterConfig';
+// Components and Pages
+import css from './css/Fund.css';
+import FundPane from './FundPane';
+import Fund from './Fund';
+
+const INITIAL_RESULT_COUNT = 30;
+const RESULT_COUNT_INCREMENT = 30;
+const filterConfig = Filters();
+const searchableIndexes = SearchableIndexes;
+
+class FundsList extends Component {
+  static propTypes = {
+    match: PropTypes.object,
+    stripes: PropTypes.object,
+    onSelectRow: PropTypes.func,
+    mutator: PropTypes.object.isRequired,
+    resources: PropTypes.object.isRequired,
+    onComponentWillUnmount: PropTypes.func
+  }
+
+  static manifest = Object.freeze({
+    initializedFilterConfig: { initialValue: false },
+    query: {
+      initialValue: {
+        query: '',
+        filters: '',
+        sort: ''
+      },
+    },
+    resultCount: { initialValue: INITIAL_RESULT_COUNT },
+    records: {
+      type: 'okapi',
+      clear: true,
+      records: 'funds',
+      recordsRequired: '%{resultCount}',
+      path: FUNDS_API,
+      perRequest: RESULT_COUNT_INCREMENT,
+      throwErrors: false,
+      GET: {
+        params: {
+          query: makeQueryFunction(
+            'cql.allRecords=1',
+            '(name="%{query.query}*")',
+            {},
+            filterConfig,
+          ),
+        },
+        staticFallback: { params: {} },
+      },
+    },
+  });
+
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.transitionToParams = transitionToParams.bind(this);
+    this.removeQueryParam = removeQueryParam.bind(this);
+  }
+
+  create = (fundData) => {
+    const { mutator } = this.props;
+    mutator.records.POST(fundData).then(newFund => {
+      mutator.query.update({
+        _path: `/finance/fund/view/${newFund.id}`,
+        layer: null
+      });
+    });
+  }
+
+  renderNavigation = () => (
+    <FinanceNavigation />
+  );
+
+  render() {
+    const { onSelectRow, onComponentWillUnmount, resources, mutator, match, stripes } = this.props;
+    const resultsFormatter = {
+      'name': data => get(data, ['name'], ''),
+      'code': data => toString(get(data, ['code'], '')),
+      'fundStatus': data => toString(get(data, ['fundStatus'], '')),
```

Comment:
For what you use `toString` and `get` together? `get` will return '' if value `undefined`. Or this value can be `null`?

Reply:
It was copy-paste from `<Fund />` due to files structure changes changes